CMakeLists.txt

```diff
@@ -22,18 +24,16 @@ if (NOT WIN32 OR MSYS OR CYGWIN)
     OUTPUT_STRIP_TRAILING_WHITESPACE
   )
 
-  # TODO: explain why these are required.
+  # Shaobo: I think we may want this flag gone because we want to enable
```

Comment:
What does this flag do? Enable assertions where? In SMACK?
I am almost certain that the assertions in SMACK were enabled even with this flag here.